tests/eval-test.sh

```diff
@@ -1,15 +1,11 @@
 #!/bin/sh
+set -efu
+cd "$(dirname "$0")/.." || exit 1
```

Comment:
This can't really fail.

Reply:
`exit 1` should not be necessary, since `set -e` is used.